provisioner/k8s/controllers/bundle_controller.go

```diff
@@ -347,7 +337,7 @@ func (r *BundleReconciler) getBundleImageDigest(pod *corev1.Pod) (string, error)
 
 func getObjects(bundleFS fs.FS) ([]client.Object, error) {
 	var objects []client.Object
-	const manifestsDir = "manifests"
+	const manifestsDir = "."
```

Comment:
I would expect this to grab from a label or other defined directory. Should we default to expecting the manifests to be in the root of a plain bundle?

Reply:
My two cents: by the time we're calling this function, `bundleFS` should be organized into a consistent view such that we don't need to parse metadata or lookup labels to interpret it.

I'd go even further and say that our plain manifest bundle format should define `/manifests` as _the_ one and only location for manifests. And we if add metadata eventually (I think we'll need to), then we should have a well-defined structure for that as well (specific directories, filenames, and formats)

Reply:
Yeah that was the original proposal a couple of quarters back, and we can even introduce tooling for statically validating the bundle format ahead of time, and ensuring that /manifests is the consistent destination container image directory.